client/components/Login.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useState } from 'react';
```

Comment:
This component is using a mix of sans-serif and serif font styles. Let's pick one for the entire app and stick with it.

Reply:
There is still a mixture of Flora's font and a default one. Will address that at some point for sure.